[From http://mirror.keystealth.org/gnu/bash/bash-5.1-patches/bash51-001]
Signed-off-by: Peter Seiderer <ps.report@gmx.net>

			     BASH PATCH REPORT
			     =================

Bash-Release:	5.1
Patch-ID:	bash51-001

Bug-Reported-by:	Fazal Majid <fazal@majid.org>
Bug-Reference-ID:	<DEAB7D2C-C626-450C-B2E5-281AFF2D26D4@majid.org>
Bug-Reference-URL:	https://lists.gnu.org/archive/html/bug-bash/2020-12/msg00000.html

Bug-Description:

There is a missing dependency on a constructed file, which can cause highly
parellel builds to fail.

Patch (apply with `patch -p0'):

*** ../bash-5.1-patched/Makefile.in	2020-12-04 09:51:19.000000000 -0500
--- b/Makefile.in	2020-12-16 11:28:36.000000000 -0500
***************
*** 1316,1319 ****
--- 1316,1320 ----
  bashline.o: pcomplete.h ${BASHINCDIR}/chartypes.h input.h
  bashline.o: ${BASHINCDIR}/shmbutil.h ${BASHINCDIR}/shmbchar.h
+ bashline.o: ${DEFDIR}/builtext.h
  bracecomp.o: config.h bashansi.h ${BASHINCDIR}/ansi_stdlib.h
  bracecomp.o: shell.h syntax.h config.h bashjmp.h ${BASHINCDIR}/posixjmp.h
***************
*** 1436,1439 ****
--- 1437,1441 ----
  builtins/evalstring.o: jobs.h builtins.h flags.h input.h execute_cmd.h
  builtins/evalstring.o: bashhist.h $(DEFSRC)/common.h pathnames.h
+ builtins/evalstring.o: ${DEFDIR}/builtext.h
  builtins/getopt.o: config.h ${BASHINCDIR}/memalloc.h
  builtins/getopt.o: shell.h syntax.h bashjmp.h command.h general.h xmalloc.h error.h

*** ../bash-5.1-patched/builtins/Makefile.in	2019-07-25 08:03:45.000000000 -0400
--- b/builtins/Makefile.in	2020-12-16 11:29:29.000000000 -0500
***************
*** 362,366 ****
  evalstring.o: $(topdir)/flags.h $(topdir)/input.h $(topdir)/execute_cmd.h
  evalstring.o: $(topdir)/bashhist.h $(srcdir)/common.h
! evalstring.o: $(topdir)/trap.h $(topdir)/redir.h ../pathnames.h
  #evalstring.o: $(topdir)/y.tab.h
  getopt.o: ../config.h $(BASHINCDIR)/memalloc.h
--- 362,366 ----
  evalstring.o: $(topdir)/flags.h $(topdir)/input.h $(topdir)/execute_cmd.h
  evalstring.o: $(topdir)/bashhist.h $(srcdir)/common.h
! evalstring.o: $(topdir)/trap.h $(topdir)/redir.h ../pathnames.h ./builtext.h
  #evalstring.o: $(topdir)/y.tab.h
  getopt.o: ../config.h $(BASHINCDIR)/memalloc.h

*** ../bash-5.1/patchlevel.h	2020-06-22 14:51:03.000000000 -0400
--- b/patchlevel.h	2020-10-01 11:01:28.000000000 -0400
***************
*** 1,5 ****
  /* patchlevel.h -- current bash patch level */
  
! /* Copyright (C) 2001-2016 Free Software Foundation, Inc.
  
     This file is part of GNU Bash, the Bourne Again SHell.
--- 1,5 ----
  /* patchlevel.h -- current bash patch level */
  
! /* Copyright (C) 2001-2020 Free Software Foundation, Inc.
  
     This file is part of GNU Bash, the Bourne Again SHell.
***************
*** 26,30 ****
     looks for to find the patch level (for the sccs version string). */
  
! #define PATCHLEVEL 0
  
  #endif /* _PATCHLEVEL_H_ */
--- 26,30 ----
     looks for to find the patch level (for the sccs version string). */
  
! #define PATCHLEVEL 1
  
  #endif /* _PATCHLEVEL_H_ */
